src/main/java/com/softdevsix/api/dtos/FileCoverageDTO.java

```diff
@@ -0,0 +1,55 @@
+package com.softdevsix.api.dtos;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.List;
+
+/**
+ * Data Transfer Object (DTO) representing the coverage details of a file.
+ * This DTO is used to transfer file coverage information.
+ */
+@Getter
+@Setter
+public class FileCoverageDTO {
```

Comment:
Why do you decide to use `class` type for DTOs?

Reply:
The use of a class for DTOs is a good practice because it provides a clear, reusable, and strongly typed structure for transferring data. Alternatives like record may offer more concise and immutable representations, but classes are still the most compatible with frameworks like Spring Boot, which heavily utilize annotations and mutability for data handling